### ISSUE TITLE
Tunny runtime error occurs when trying to run optimization in human-in-the-loop mode

### ISSUE THREAD

**Issue by toniramo** [hrntsm/Tunny#327]
### Expected behavior

Runtime errors should not occur, and the 'Which trial is the worst' dashboard should display results once the first batch is processed.

### Environment

- OS: Windows 10 Enterprise (Build 19045)
- Rhino version: 8 SR11
- Tunny version: 0.12.0

### Steps to reproduce

1. Open the example file [Karamba3d_column_position_opt.gh](https://github.com/hrntsm/Tunny/blob/main/Examples/Grasshopper/Human-in-the-loop/Karamba3d_column_position_opt.gh)
2. Double-click on the Tunny component
3. Click the RunOptimize button.
4. Wait for the result.

### Error messages, stack traces, or logs

After the first batch is processed and the Optuna dashboard opens, an error message is displayed:
![image](https://github.com/user-attachments/assets/a585ca40-c2aa-4031-a95c-5b9c26aec0ae)

The 'Which trial is the worst' dashboard does not display any results:
![image](https://github.com/user-attachments/assets/00e9ad16-46f8-44a6-adfd-38f43ab7bae9)

[log_20241009.txt](https://github.com/user-attachments/files/17309009/log_20241009.txt)

**Comment by hrntsm** [comment]
@toniramo 
Thanks for the report. I will check it out.

**Comment by hrntsm** [comment]
@toniramo 
I have fixed it. Can you try the corrected version below?
https://github.com/hrntsm/Tunny/actions/runs/11269248296/artifacts/2038834769

Perhaps you are the one who emailed me about the user profile issue?
If so, the fix is reflected in the develop branch, so please build that one.

**Comment by hrntsm** [comment]
I found another bug and will fix it as well. Please wait a little.

**Comment by hrntsm** [comment]
@toniramo
I have fixed it. 
Could you try the corrected version below or develop branch?
https://github.com/hrntsm/Tunny/actions/runs/11271381198/artifacts/2039396137

**Comment by toniramo** [comment]
@hrntsm 
Yes, I did contact you earlier. (I resolved the rights issue with IT support, although the alternative approach also worked.) 

After testing with the updated version, no _stop_flag error occurred, and the dashboard with the first 6 trials appears as expected (see the attached recording). :+1: Thank you for solving the problem.

However, I noticed that sometimes after selecting the worst trials, the corresponding boxes may end up stuck on the dashboard, though that must be a separate issue.

https://github.com/user-attachments/assets/504bc286-380a-4f95-8d05-fc9d5386436a
